project-base/storefront/components/Pages/Order/PaymentConfirmation/Gateways/GoPayGateway.tsx

```diff
@@ -38,21 +46,20 @@ export const GoPayGateway: FC<GoPayGatewayProps> = ({ orderUuid }) => {
         });
     };
 
-    if (!goPayPaymentSetup) {
-        return null;
-    }
-
     return (
         <>
-            <Script
-                id="go-pay-embedded-js"
-                src={goPayPaymentSetup.embedJs}
-                onLoad={initGoPayCheckout(goPayPaymentSetup.gatewayUrl)}
-            />
-            {isRedirectLinkVisible && (
-                <Link isButton isExternal href={goPayPaymentSetup.gatewayUrl}>
-                    {t('Pay with GoPay')}
-                </Link>
+            {initiatedPaymentGate && !!goPayPaymentSetup && (
+                <Script
+                    id="go-pay-embedded-js"
+                    src={goPayPaymentSetup.embedJs}
+                    onLoad={initGoPayCheckout(goPayPaymentSetup.gatewayUrl)}
+                />
+            )}
+            {requiresAction && (
```

Comment:
Tiny thing. Can we separate this with one line, please? Since all FE agreed on this would be great if you would have it on your mind also for other places. Visibility is much better, at least for most of us.

Reply:
Done